pkg/api/rest/resttest/resttest.go

```diff
@@ -961,7 +961,7 @@ func (t *Tester) testWatchFields(obj runtime.Object, initWatchFn InitWatchFunc,
 				if !ok {
 					t.Errorf("watch channel should be open")
 				}
-			case <-time.After(time.Millisecond * 100):
+			case <-time.After(time.Second * 30):
```

Comment:
No, its not, I've seen this flake, and again, this only happens when the test is broken.  There's no reason at all to sleep effectively forever here.  Most of the time, the case above this timeout will trigger way before anything else, and this is just hear to catch failures.